### PR TITLE
Add cisagov/disable-inactive-iam-users-tf-module

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds the [cisagov/disable-inactive-iam-users-tf-module](https://github.com/cisagov/disable-inactive-iam-users-tf-module) Terraform module to periodically disable inactive IAM users.

This module creates an EventBridge event rule that triggers at a fixed cadence, kicking off a Lambda function that disables inactive IAM users.

See also cisagov/cool-accounts#147.

## 💭 Motivation and context ##

We have a POAM about disabling inactive users in IAM.  Partially resolves cisagov/cool-system-internal#143.

## 🧪 Testing ##

I applied these changes to both staging and production without incident.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,10 +122,14 @@ future changes by simply running `terraform apply
...
 | provisionssmsessionmanager\_policy\_description | The description to associate with the IAM policy that allows sufficient permissions to provision the SSM Document resource and set up SSM session logging in the User Services account. | `string` | `"Allows sufficient permissions to provision the SSM Document resource and set up SSM session logging in the User Services account."` | no |
 | provisionssmsessionmanager\_policy\_name | The name to assign the IAM policy that allows sufficient permissions to provision the SSM Document resource and set up SSM session logging in the User Services account. | `string` | `"ProvisionSSMSessionManager"` | no |
+| read\_lambda\_bucket\_policy\_description | The description to associate with the IAM role that allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments. | `string` | `"Allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments."` | no |
```
Comment (dav3r) [inline README.md]: Remove duplicated words:
```suggestion
| read\_lambda\_bucket\_policy\_description | The description to associate with the IAM role that allows read-only access to the bucket in the Terraform account containing Lambda deployments. | `string` | `"Allows read-only access to the bucket in the Terraform account containing Lambda deployments."` | no |
```

File: read_lambda_bucket_policy.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows all of the permissions necessary
+# to read from the bucket containing the COOL lambda deployments.
```
Comment (dav3r) [inline read_lambda_bucket_policy.tf]: Capitalization:
```suggestion
# to read from the bucket containing the COOL Lambda deployments.
```

File: variables.tf [hunk 3]
```diff
@@ -34,6 +50,18 @@ variable "provisionssmsessionmanager_policy_name" {
   type        = string
 }
 
+variable "read_lambda_bucket_policy_description" {
+  default     = "Allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments."
+  description = "The description to associate with the IAM role that allows read-only access read-only access to the bucket in the Terraform account containing Lambda deployments."
```
Comment (dav3r) [inline variables.tf]: Remove duplicated words:
```suggestion
  default     = "Allows read-only access to the bucket in the Terraform account containing Lambda deployments."
  description = "The description to associate with the IAM role that allows read-only access to the bucket in the Terraform account containing Lambda deployments."
```

**Review by dav3r — APPROVED** [review]
👍

**Review by dv4harr10 — APPROVED** [review]

File: variables.tf (hunk 3 above)
Comment (jsf9k) [reply variables.tf]: Please see commit [22984eb](https://github.com/cisagov/cool-accounts-userservices/pull/39/commits/22984ebcb00b4bebaf7af0200496d4012e163e56).

File: read_lambda_bucket_policy.tf (hunk 2 above)
Comment (jsf9k) [reply read_lambda_bucket_policy.tf]: Please see commit [22984eb](https://github.com/cisagov/cool-accounts-userservices/pull/39/commits/22984ebcb00b4bebaf7af0200496d4012e163e56).

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Please see commit [22984eb](https://github.com/cisagov/cool-accounts-userservices/pull/39/commits/22984ebcb00b4bebaf7af0200496d4012e163e56).

**Pull request merged**